tests/CodeSnifferRunner.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ObjectCalisthenics\Tests;
+
+use PHP_CodeSniffer;
+
+final class CodeSnifferRunner
+{
+    /**
+     * @var PHP_CodeSniffer
+     */
+    private $codeSniffer;
+
+    public function __construct()
+    {
+        $this->codeSniffer = new PHP_CodeSniffer();
+    }
+
+    /**
+     * @param string $testedFile
+     * @param string $sniffName
+     *
+     * @return int
+     */
+    public function detectErrorCountInFileForSniff($testedFile, $sniffName)
+    {
+        $this->codeSniffer->initStandard('../../../../../src/ObjectCalisthenics', [$sniffName]);
```

Comment:
Yes, you can replace that relative path with `__DIR__ . '/../src/ObjectCalisthenics'`